app/v2/controllers/requestParsers/validators/validations/JsonFormatValidation.scala

```diff
@@ -16,18 +16,47 @@
 
 package v2.controllers.requestParsers.validators.validations
 
+import play.api.Logger
 import play.api.libs.json._
-import v2.models.errors.MtdError
+import v2.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
 
 object JsonFormatValidation {
```

Comment:
This file has 0% coverage.

Reply:
This file is unused anywhere in the application, thanks for finding